### DISCUSSION TITLE
8C2DE41B-5D8D-710F-547A-3C6F774CDEB4

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#32] [General]
# Planet Nine

Planet Nine is a hypothetical planet in the outer region of the Solar System. Their gravitational effects could explain the peculiar clustering of orbits of a group of extreme Trans-Neptunian Objects (ETNOs). If it does exist, Planet Nine is estimated to be an ice giant (similar to Neptune or Uranus) and is 10 times more massive than Earth. It is estimated to have an orbital period of 10,000 to 20,000 Earth years.

https://celestia.mobi/addon?item=8C2DE41B-5D8D-710F-547A-3C6F774CDEB4&lang=en

**Comment by exocritic5457** [comment]
Any nemesis addons reaching mobi? So that all of those nemesis believers will finally stop

**Reply by DaveBowman2001** [reply]
There's a Nibiru addon published by Gurren Lagann in the Celestia Forums on the 10th anniversary of the "2012 Phenomenon", but so far it hasn't yet uploaded here on mobi

**EDIT (251228)**
It's now published on mobi since [November 2023](https://celestia.mobi/resources/item?item=A5FD601A-5ADD-4B9E-A25B-835B597543FD)

**Comment by dalmi38** [comment]
should this be updated Planet nine too small now to be an ice giant

**Reply by dalmi38** [reply]
*this should

**Comment by zjx29rjhdh-pixel** [comment]
Now we have Nibiru, but Planet Nine looks like Neptune . It hasn’t rings and/or moons. It has a lack of texture.
